decoder/src/logger/mod.rs

```diff
@@ -263,3 +264,105 @@ fn color_diff(text: String) -> String {
     // keep output as it is
     text.bold().to_string()
 }
+
+/// Initializes a `log` sink that handles defmt frames.
+///
+/// Defmt frames will be printed to stdout, other logs to stderr.
+///
+/// The caller has to provide a `should_log` closure that determines whether a log record should be
+/// printed.
+///
+/// If `always_include_location` is `true`, a second line containing location information will be
+/// printed for *all* records, not just for defmt frames.
```

Comment:
nit: from these docs I would've interpreted that `always_include_location = false` deactivates locations from `defmt` logs as well but from the `log()` implementation it seems that it is always on?

Reply:
Clarified the docs a bit